locales/en-US/common.ftl

```diff
@@ -10,6 +10,7 @@ production-use = Production use
 learn-more = Learn More
 discord = Discord
 zulip = Zulip
+mastodon = Mastodon
```

Comment:
This needs to go in every Fluent file, translators should have the ability to tweak it if they wish.